index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
```

Comment:
can you put all your code in a `project_one` folder? this file should show up in the diff as `project_one/index.html`